docker-compose.test.yml

```diff
@@ -195,43 +195,43 @@ services:
         NCCL_VERSION_OVERRIDE: 2.7.8-1+cuda10.1
         MPI_KIND: OpenMPI
         PYTHON_VERSION: 3.6
-        TENSORFLOW_PACKAGE: tensorflow-gpu==2.2.0
+        TENSORFLOW_PACKAGE: tensorflow-gpu==2.3.1
         KERAS_PACKAGE: keras==2.3.1
-        PYTORCH_PACKAGE: torch==1.5.0+cu101
-        TORCHVISION_PACKAGE: torchvision==0.6.0+cu101
+        PYTORCH_PACKAGE: torch==1.6.0+cu101
+        TORCHVISION_PACKAGE: torchvision==0.7.0+cu101
         MXNET_PACKAGE: mxnet-cu101==1.6.0.post0
         PYSPARK_PACKAGE: pyspark==2.4.7
         SPARK_PACKAGE: spark-2.4.7/spark-2.4.7-bin-hadoop2.7.tgz
-  test-gpu-openmpi-py3_6-tf2_3_0-keras2_3_1-torch1_6_0-mxnet1_6_0-pyspark2_4_7:
+  test-gpu-openmpi-gloo-py3_6-tf2_4_0-keras2_3_1-torch1_7_1-mxnethead-pyspark2_4_7:
     extends: test-gpu-base
     build:
       args:
-        CUDA_DOCKER_VERSION: 10.1-devel-ubuntu18.04
-        CUDNN_VERSION: 7.6.5.32-1+cuda10.1
-        NCCL_VERSION_OVERRIDE: 2.7.8-1+cuda10.1
+        CUDA_DOCKER_VERSION: 11.0-devel-ubuntu18.04
+        CUDNN_VERSION: 8.0.5.39-1+cuda11.0
+        NCCL_VERSION_OVERRIDE: 2.7.8-1+cuda11.0
         MPI_KIND: OpenMPI
         PYTHON_VERSION: 3.6
-        TENSORFLOW_PACKAGE: tensorflow-gpu==2.3.0
+        TENSORFLOW_PACKAGE: tensorflow-gpu==2.4.0
         KERAS_PACKAGE: keras==2.3.1
-        PYTORCH_PACKAGE: torch==1.6.0+cu101
-        TORCHVISION_PACKAGE: torchvision==0.7.0+cu101
-        MXNET_PACKAGE: mxnet-cu101==1.6.0.post0
+        PYTORCH_PACKAGE: torch==1.7.1+cu110
+        TORCHVISION_PACKAGE: torchvision==0.8.2+cu110
+        MXNET_PACKAGE: mxnet-cu110
         PYSPARK_PACKAGE: pyspark==2.4.7
         SPARK_PACKAGE: spark-2.4.7/spark-2.4.7-bin-hadoop2.7.tgz
   test-gpu-openmpi-py3_6-tfhead-kerashead-torchhead-mxnethead-pyspark2_4_7:
     extends: test-gpu-base
     build:
       args:
-        CUDA_DOCKER_VERSION: 10.1-devel-ubuntu18.04
-        CUDNN_VERSION: 7.6.5.32-1+cuda10.1
-        NCCL_VERSION_OVERRIDE: 2.7.8-1+cuda10.1
+        CUDA_DOCKER_VERSION: 11.0-devel-ubuntu18.04
+        CUDNN_VERSION: 8.0.5.39-1+cuda11.0
+        NCCL_VERSION_OVERRIDE: 2.7.8-1+cuda11.0
         MPI_KIND: OpenMPI
         PYTHON_VERSION: 3.6
         TENSORFLOW_PACKAGE: tf-nightly-gpu
         KERAS_PACKAGE: keras==2.4.3
         PYTORCH_PACKAGE: torch-nightly
         TORCHVISION_PACKAGE: torchvision
-        MXNET_PACKAGE: mxnet-nightly
+        MXNET_PACKAGE: mxnet-cu110
```

Comment:
is this still a head version of mxnet?

Reply:
It is, yeah.  Possibly we could change this `mxnet-nightly-cu110` if it's more clear.